.github/workflows/ci.yml

```
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby_version: ['3.1', '3.0', '2.7', '2.6.10', '2.6.1']
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby_version }}
+
+    - name: Bundle install
+      run: |
+        bundle check || bundle install
```

Why not using `bundler-cache` on `ruby/setup-ruby` action?

```suggestion
      uses: ruby/setup-ruby@v1
      with:
        ruby-version: ${{ matrix.ruby_version }}
        bundler-cache: true
```

---

I just forgot it. Thanks!